cli/src/common_utils/lr-serial-number

```diff
@@ -29,10 +29,10 @@ function lr_serial_number {
     else
         if [[ -f /opt/seagate/lr-serial-number ]]; then
             echo "Lyve Rack r1 Serial Number already set."
-            exit 31
+            exit 0
```

Comment:
The non-zero exit here is on purpose.
The caller can decide to neglect it.
The primary purpose of this script is to set the value.